### PR TITLE
Update EULA tooltip in Developer Hub

### PR DESCRIPTION
**Pull request by rbrishabh**
Fixes 1044 of https://github.com/mozilla/addons/issues/1044.
@eviljeff @caitmuenster @EnTeQuAk 

Let me know if this needs any changes! Thanks!

Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.

Please delete anything that isn't relevant to your patch.

* [x] This PR relates to an existing open issue and there are no existing
      PRs open for the same issue.
* [x] Add `Fixes #ISSUENUM` at the top of your PR.
* [x] Add a description of the changes introduced in this PR.
* [x] The change has been successfully run locally.
* [x] Add tests to cover the changes added in this PR.
* [x] Add before and after screenshots (Only for changes that impact the UI).

Once you have met the above requirements please replace this section with
a `Fixes #ISSUENUM` linking to the issue fixed by this PR along with an
explanation of the changes. Thanks for your contribution!

### CODE REVIEW

File: src/olympia/devhub/templates/devhub/includes/policy_form.html [hunk 1]
```diff
@@ -1,11 +1,9 @@
 {% from "devhub/includes/macros.html" import tip, some_html_tip %}
 <tr>
   <th>{{ tip(_('End-User License Agreement'),
-             _('Users will be required to accept the following End-User License Agreement (EULA) '
-               'prior to installing your add-on. The presence of a EULA significantly affects '
-               'the number of downloads an add-on receives. Please note that a EULA is not '
+             _('Please note that a EULA is not '
                'the same as a code license, such as the GPL or MPL.'
-               'It is only relevant for listed add-ons.')) }}</th>
+               ' It is only relevant for listed add-ons.')) }}</th>
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/includes/policy_form.html]: minor point, but we always have the space on the end of the previous string (so after the `MPL.`)

Comment (rbrishabh) [reply src/olympia/devhub/templates/devhub/includes/policy_form.html]: @eviljeff Alright, got it! Thanks

**Comment by caitmuenster** [comment]
Thanks so much for the patch, @rbrishabh! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Welcome onboard! We look forward to seeing you around.

**Comment by rbrishabh** [comment]
@eviljeff No problem man, great learning experience for me!

@caitmuenster, you're welcome! Thanks for the recognition. I look forward to seeing you around as well :)

**Pull request merged**